test/parallel/test-assert-checktag.js

```diff
@@ -45,7 +45,8 @@ function re(literals, ...values) {
   }
   assert.doesNotThrow(() => assert.deepEqual(fakeGlobal, global));
   // Message will be truncated anyway, don't validate
-  assert.throws(() => assert.deepStrictEqual(fakeGlobal, global));
+  assert.throws(() => assert.deepStrictEqual(fakeGlobal, global),
+                /^AssertionError: /);
```

Comment:
As above, consider just passing the `AssertionError` constructor here instead of a regexp.